lib/views/pages/new_post/new_post_form.dart

```diff
@@ -92,7 +78,12 @@ class NewPostForm extends HookConsumerWidget {
               fit: FlexFit.loose,
               child: _verticallyPadded(bodyField),
             ),
-            _verticallyPadded(buttonRow),
+            _verticallyPadded(
+              SizedBox(
```

Comment:
This used to be an `Expanded`. By humble curiosity, why did you choose to use a `SizedBox` of infinite width instead?